build/configs/artik053/README.md

```diff
@@ -91,41 +91,24 @@ openocd -f artik053.cfg -c ' \
 
 ## ROMFS
 
-Before executing below steps, execute [generic steps](../../../tools/fs/README_ROMFS.md), step 1 and step 2.
+Execute [generic steps](../../../tools/fs/README_ROMFS.md), step 1 and step 2.
+If you want to change the size of ROMFS partition, execute below steps.
```

Comment:
If you want to change a line like you wrie in this file, you should add two spaces at end of previous line.

Reply:
Ok. I need to modify it.

Reply:
Ok. I need to modify it.